doc/admin-guide/files/records.config.en.rst

```diff
@@ -3884,6 +3884,12 @@ QUIC Configuration
 All configurations for QUIC are still experimental and may be changed or
 removed in the future without prior notice.
 
+.. ts:cv:: CONFIG proxy.config.quic.qlog_dir STRING nullptr
```

Comment:
It seems like we use `NULL` on the documentation. Maybe `nullptr` is too C++ specific.

Reply:
fixed